src/table/utils/handle-click.ts

```diff
@@ -39,9 +39,9 @@ export const getSelectionMouseHandlers = (
   announce: Announce,
   onMouseDown: React.MouseEventHandler<HTMLTableCellElement> | undefined,
   selectionDispatch: SelectionDispatch,
-  isFlagEnabled: (flag: string) => boolean
+  areBasicFeaturesEnabled: boolean
 ) => {
-  const selectMultiEnabled = isFlagEnabled('PS_15585_SN_TABLE_BASIC_FEATURES');
+  const selectMultiEnabled = areBasicFeaturesEnabled;
```

Comment:
might as well use the `areBasicFeaturesEnabled` in the code

Reply:
Ah, so silly! yes off course, I'll fix it.